src/main/java/com/nakaligoba/backend/member/application/MemberService.java

```diff
@@ -43,18 +45,20 @@ public class MemberService {
     private final RedisUtils redisUtils;
 
     @Transactional
-    public boolean signup(MemberDto memberDto) {
-        if (!memberRepository.existsByEmail(memberDto.getEmail())) {
+    public String signup(MemberDto memberDto) {
+        if (memberRepository.existsByEmail(memberDto.getEmail())) {
+            return SignupResponse.DUPLICATE_EMAIL;
+        } else if (memberRepository.existsByNickname(memberDto.getNickname())) {
+            return SignupResponse.DUPLICATE_NICKNAME;
+        } else {
```

Comment:
분기문에서 바로 return을 하므로 다음과 같이 작성할 수 있습니다! 가독성도 좋아지구요!
```suggestion
        if (memberRepository.existsByEmail(memberDto.getEmail())) {
            return SignupResponse.DUPLICATE_EMAIL;
        if (memberRepository.existsByNickname(memberDto.getNickname())) {
            return SignupResponse.DUPLICATE_NICKNAME;
        }
```